src/main/java/uk/gov/hmcts/reform/et/syaapi/service/NotificationService.java

```diff
@@ -284,12 +294,17 @@ public void sendAcknowledgementEmailToRespondents(CaseData caseData,
             caseNumber
         );
         respondentParameters.put(
-            HEARING_DATE,
+            SEND_EMAIL_PARAMS_HEARING_DATE_KEY,
             hearingDate
         );
-        respondentParameters.put("shortText",
-                                 APP_TYPE_MAP.get(claimantApplication.getContactApplicationType()));
-        respondentParameters.put("datePlus7", LocalDate.now().plusDays(7).toString());
+        respondentParameters.put(
+            SEND_EMAIL_PARAMS_SHORTTEXT_KEY,
+            APP_TYPE_MAP.get(claimantApplication.getContactApplicationType())
+        );
+        respondentParameters.put(
+            SEND_EMAIL_PARAMS_DATEPLUS7_KEY,
+            LocalDate.now().plusDays(7).toString()
```

Comment:
LocalDate.now().plusDays(7).toString() is used twice can it be added to a varaible